src/wisb/particle_data.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <array>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "vector.hpp"
+
+namespace wash {
+
+    void _initialise_particle_data(size_t particlec);
+
+    extern std::vector<SimulationVecT>* vector_force_pos;
+    extern std::vector<SimulationVecT>* vector_force_vel;
+    extern std::vector<SimulationVecT>* vector_force_acc;
```

Comment:
Is unique_ptr heap allocation? I didn't think leaks were a super big issue since these data have a static program lifetime but it should be a simple enough change

Reply:
Yes its heap - `unique_ptr`s just wrap raw pointers but handle deallocation when it goes out of scope.

In this instance it's not a big deal, but in other instances this can lead to huge issues (with e.g. accessing memory that has already been deleted by another function).

Surely these don't need to be pointers anyway?

Reply:
I think it's easier if we just make the force vectors global variables - that's what I did on my branch.

Also, Cornerstone requires all force vectors to be of the same type (e.g. double), so I've split each vector force into 3 scalar forces. But I guess we can fix that when merging Cornerstone branch.

Reply:
Need to be ptrs to I think allow for dynamic initialisation to `particlec` simulation parameter. Made unique_ptrs in latest commit however.